app/views/posts/index.html.erb

```diff
@@ -1,5 +1,4 @@
 <div class="u-container">
-<h1>Displays user posts</h1>
 <img class="u_image" src="<%= @user.photo ? @user.photo : 'https://images.unsplash.com/photo-1582567056798-7dc94989e56d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80' %>">
     <div class="u-info">
     <h3><%= link_to @user.name, @user %></h3>
```

Comment:
- [ ] @Dadadon Great work on your post index page, however, the positioning of the posts counter and pagination is wrong, also doesn't display the counter and likes counter and comments for each post, kindly check the comparison below and ensure your design matches the design given:

Project Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Dadadon/blog/assets/61408860/31796105-829d-49d0-afbe-17609a80bea0)  | ![image](https://github.com/Dadadon/blog/assets/61408860/5da31721-1c5a-4e79-b44b-8dda12702612)

Reply:
- [ ] @Dadadon Great work on your post-show page, however, the positioning of the counter and likes counter is wrong, also ensure your title matches the design of the title given, kindly check the comparison below and ensure your design matches the design given:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Dadadon/blog/assets/61408860/037d0695-b1a2-4ee0-92f4-976621c2e9cf)  | ![image](https://github.com/Dadadon/blog/assets/61408860/9397c061-6dd2-4d9e-bdec-952becf5df8c)